sdks/wasm/src/library_mono.js

```diff
@@ -613,6 +618,12 @@ var MonoSupportLib = {
 			return this.mono_wasm_setup_single_step (kind);
 		},
 
+		mono_wasm_set_pause_on_exceptions: function (state) {
```

Comment:
Maybe we should pass in a string `none`/`uncaught`/`all`?

Reply:
And `library_mono.js` can convert that to the expected enum, for `*debugger.c`.

Reply:
done.